### DISCUSSION TITLE
setting up dual monitor applications to open on two workspaces

### DISCUSSION THREAD

**Discussion by stibinator** [i3/i3#6322] [Q&A] [answered]
I use Blender and DaVinci Resolve on a dual monitor system. Both of these programs are able to work with two main windows open, one on each monitor, fullscreen. To set this up manually I have to:
 - run the program, 
 - go to the left window, send it to a new workspace on my left monitor so it is full screen
 - switch to the right window and send it to a new workspace on my right monitor, full screen
 - switch to the new left and right workspaces
Is there a way that I can set this up in my .conf so it happens each time I run the programs? I understand that I can assign the client to a workspace, but I don't know how to assign it to *two* workspaces.

**Comment by orestisfl** [comment, marked as answer]
This should be generally possible with the [IPC interface](https://i3wm.org/docs/ipc.html).

You can create a script that:

1. Watch for new window events
2. Start the two programs
3. When window A starts, send it to the left monitor, fullscreen
4. When window B starts, send it to the right monitor, fullscreen
5. Exit

**Reply by stibinator** [reply]
Thanks, I've got a huge list of things to configure after switching to i3wm, I'll add learning the IPC interface to the list

**Reply by TheTechRun** [reply]
Or you could use i3wm commands to move specific windows to specific workspaces based on title.

**Reply by stibinator** [reply]
I have Blender assigned to a workspace which is assigned to one monitor, and I added a bindsym that opens Blender, then opens a new workspace that I've mapped to the other monitor. I then have to manually send one of the blender windows to this new workspace, because they both open on the same one. The problem is that all the blender windows have the same name, there's no way of telling one from another.

**Reply by stibinator** [reply]
Resolve has a splash screen, then a project manager screen, and only once you've chosen a project does it open two screens if you've set it to be in dual monitor mode. So for that I just have to put up with manually arranging it each time.
